src/interactions/pageView.ts

```diff
@@ -10,12 +10,8 @@ type PageViewOptions = {
 
 export function pageView(
   { title, location, path, sendPageView, userId }: PageViewOptions = {},
-  measurementId?: string
 ): void {
-  const gaMeasurementId =
-    process.env.NEXT_PUBLIC_GA_MEASUREMENT_ID ?? measurementId;
-
-  if (!gaMeasurementId || !window.gtag) {
+  if (!window.gtag) {
```

Comment:
Thanks for the update @Cretezy .

I saw [your comment on the `event.ts`](https://github.com/MauricioRobayo/nextjs-google-analytics/pull/313#discussion_r1023264784) file and thought you were referring only to that file. That's a great catch!

Here, on `pageView.ts` the `gaMeasurementId` is needed as it is required by `window.gtag("config", gaMeasurementId, pageViewOptions);` at the end of the file.

I apologize I misunderstood your suggestion.

If possible, could you revert `gaMeasurementId` on this file.

Reply:
You're totally right, sorry missed that.

Why not just use the page_view event directly instead of config?
https://developers.google.com/tag-platform/gtagjs/reference/events#page_view

This has the same options (page_title, page_location, path_path, user_id), except send_page_view (which is kind of redundant here right? Setting that to false currently makes this a no-op?).

Reply:
Hello @Cretezy !

> Why not just use the page_view event directly instead of config?

great question! At the beginning the idea was for `pageview` to be reusable if someone wants more grain control, so it is also exported, and anyone can create a custom hook with it.

There might be cases in which someone might want to opt-out of the default behavior provided by the library and disable page views for specific pages, and hence they could set `send_page_view` to false: https://developers.google.com/analytics/devguides/collection/ga4/views?technology=websites#disable_pageviews